docker/README.md

```diff
@@ -0,0 +1,105 @@
+# [Devito] docker image library
+
+In order to facilitate the dissemination, usage, and development of Devito, we provide a series of docker images. These images support numerous architectures and compilers and are tagged accordingly. In the following, we describe the available images and the workflow to build it yourself. You can find all the available images at [DevitoHub](https://hub.docker.com/r/devitocodes/). 
+
+## [Devito] images
+
+Devito provides three main images that are targeting different architectures and/or using different compilers. In the following, all images are described as `imagename-*`. The `*` corresponds to, and should be swapped for, the different release tags `dev`, `latest` or `vx.x.x` depending on if you are interested in specific versions (`vx.x.x`), the latest stable release(`latest`), or the latest development status (`dev`)
+
+### [Devito] on CPU
+
+We provide two CPU images:
+- `devito:gcc-*` with the standard GNU gcc compiler.
+- `devito:icc-*` with the intel C compiler for intel architectures.
+
+these base images provide a working [Devito] environment for any CPU architecture and come with [Devito], `gcc/icc` and `mpi` preinstalled as well as utilities such as `jupyter` for usability and exploration of the package.
+
+To run this image locally, you will need `docker` to be installed. Once available, the following commands will get you started:
+
+```bash
+docker run --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786 devitocodes/devito:gcc-latest
+docker run --rm -it -p 8888:8888 -p 8787:8787 -p 8786:8786 --device=/dev/infiniband/uverbs0 --device=/dev/infiniband/rdma_cm devitocodes/devito:gcc-latest
```

Comment:
only keep the former?

the latter should belong to a future section where we describe more advanced usages, such us mpi over infiniband etc.

Reply:
That'd be a bit of a pain because would;d have to go over CPU,GPU,AMD.... all over again. Now it's self contained for each image which I like a bit better

Reply:
ok